FRAuth/FRAuth/Callbacks/NumberAttributeInputCallback.swift

```diff
@@ -0,0 +1,28 @@
+// 
+//  NumberAttributeInputCallback.swift
+//  FRAuth
+//
+//  Copyright (c) 2020 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+
+/**
+ NumberAttributeInputCallback is a representation of OpenAM's NumberAttributeInputCallback to collect double value with OpenAM validation and given policies.
+ */
+@objc(FRNumberAttributeInputCallback)
+public class NumberAttributeInputCallback: AttributeInputCallback {    
+    
+    /// Sets Double input value for NumberAttributeInputCallback.
+    /// - Parameter val: Double input value for Callback
+    /// - Returns: Boolean indicator whether or not it was successful
+    @objc(setNumberValue:)
+    @discardableResult public func setValue(_ val: Double) -> Bool {
```

Comment:
Android implementation is not returning a boolean indicating the success of the operation. Do you think it is really necessary? Should we align the behaviour?

Reply:
You are correct, this discardableResult of setValue methods in Callback were designed for the other approach that I was thinking, and it was not removed. There is no point with the current design to return the boolean result of the operation. Thanks for pointing that out; removed the methods, and committed.